src/nrniv/CMakeLists.txt

```diff
@@ -444,6 +451,21 @@ if(NRN_ENABLE_PROFILING)
   target_link_libraries(nrniv_lib ${likwid_LIBRARIES} ${CALIPER_LIB})
 endif()
 
+if(NRN_ENABLE_MORPHIO)
+  find_package(MorphIO REQUIRED)
+  set(HDF5_NO_FIND_PACKAGE_CONFIG_FILE ON)
+  find_package(HDF5 REQUIRED COMPONENTS CXX)
+  target_link_libraries(nrniv_lib ${HDF5_LIBRARIES})
```

Comment:
HDF5 now also comes with targets, and IIRC `HDF5_LIBRARIES` briefly pointed at targets for a few CMake versions, leading to great confusion.

Reply:
to be checked

Reply:
I think that's why he added the `set(HDF5_NO_FIND_PACKAGE_CONFIG_FILE ON)`.